### PR TITLE
feat(components/help-inline): update documentation and deprecate `indicators/help-inline`

### PR DESCRIPTION
**Pull request by Blackbaud-SandhyaRajasabeson**
[#AB2915542](https://dev.azure.com/blackbaud/Products/_boards/board/t/SKY%20UX%20Program/Stories/?workitem=2915542)

### CODE REVIEW

**Comment by blackbaud-sky-build-user** [comment]
[Storybook preview](https://blackbaud.github.io/skyux-pr-preview/2428/storybook/)

Component Storybooks:

- [action-bars-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/action-bars-storybook/)
- [ag-grid-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/ag-grid-storybook/)
- [angular-tree-component-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/angular-tree-component-storybook/)
- [avatar-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/avatar-storybook/)
- [colorpicker-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/colorpicker-storybook/)
- [data-manager-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/data-manager-storybook/)
- [datetime-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/datetime-storybook/)
- [errors-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/errors-storybook/)
- [flyout-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/flyout-storybook/)
- [forms-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/forms-storybook/)
- [help-inline-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/help-inline-storybook/)
- [indicators-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/indicators-storybook/)
- [inline-form-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/inline-form-storybook/)
- [layout-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/layout-storybook/)
- [lists-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/lists-storybook/)
- [lookup-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/lookup-storybook/)
- [modals-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/modals-storybook/)
- [pages-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/pages-storybook/)
- [phone-field-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/phone-field-storybook/)
- [progress-indicator-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/progress-indicator-storybook/)
- [split-view-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/split-view-storybook/)
- [tabs-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/tabs-storybook/)
- [text-editor-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/text-editor-storybook/)
- [theme-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/theme-storybook/)
- [tiles-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/tiles-storybook/)
- [toast-storybook](https://blackbaud.github.io/skyux-pr-preview/2428/storybooks/toast-storybook/)

Apps:

- [code-examples](https://blackbaud.github.io/skyux-pr-preview/2428/code-examples/)
- [integration](https://blackbaud.github.io/skyux-pr-preview/2428/integration/)
- [playground](https://blackbaud.github.io/skyux-pr-preview/2428/playground/)

File: apps/playground/src/app/components/help-inline/help-inline.module.ts [hunk 1]
```diff
@@ -1,7 +1,7 @@
 import { CommonModule } from '@angular/common';
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
-import { SkyHelpInlineModule } from '@skyux/help-inline';
+import { SkyHelpInlineModule } from '@skyux/indicators';
```
Comment (Blackbaud-SteveBrush) [inline apps/playground/src/app/components/help-inline/help-inline.module.ts]: Could you make a new route for `help-inline-legacy` that uses the deprecated help inline? I think we wanted this route to use the new one.

File: libs/components/indicators/src/lib/modules/help-inline/help-inline.component.spec.ts [hunk 2]
```diff
@@ -24,20 +25,35 @@ describe('Help inline component', () => {
...
   });
 
+  it('should log that the component is deprecated', () => {
+    fixture.detectChanges();
+    expect(deprecatedLogSpy).toHaveBeenCalledWith(
+      'SkyHelpInlineComponent',
+      Object({
+        deprecationMajorVersion: 10,
+        replacementRecommendation:
+          'Use the help inline button component in the `@skyux/help-inline` library instead.',
```
Comment (Blackbaud-SteveBrush) [inline libs/components/indicators/src/lib/modules/help-inline/help-inline.component.spec.ts:52]: We're calling out the new `@skyux/help-inline` package, but all of its components/types are still marked as "internal".

**Review by Blackbaud-SteveBrush — APPROVED** [review]

**Comment by blackbaud-sky-build-user** [comment]
Cherry-pick successful! :cherries: :tada: See https://github.com/blackbaud/skyux/pull/2441

**Pull request merged**